src/sapling/saplingscriptpubkeyman.cpp

```diff
@@ -354,6 +354,33 @@ bool SaplingScriptPubKeyMan::IsSaplingNullifierFromMe(const uint256& nullifier)
         wallet->mapWallet.count(mapSaplingNullifiersToNotes.at(nullifier).hash);
 }
 
+void SaplingScriptPubKeyMan::GetSaplingNoteWitnesses(std::vector<SaplingOutPoint> notes,
```

Comment:
(866e25fc2c029bc8381fec7e1de3d2a01f047141)
can pass a const reference to `notes`.